### ISSUE TITLE
Healthcheck does not work

### ISSUE THREAD

**Issue by alexrudd2** [thirdgen88/mssql-docker#9]
```
docker inspect db | grep health
                "Status": "unhealthy",
                        "Output": "bash: /healthcheck.sh: No such file or directory\n"
                        "Output": "bash: /healthcheck.sh: No such file or directory\n"
                        "Output": "bash: /healthcheck.sh: No such file or directory\n"
                        "Output": "bash: /healthcheck.sh: No such file or directory\n"
                        "Output": "bash: /healthcheck.sh: No such file or directory\n"
                    "/healthcheck.sh"
```

In the container all appears well.
```
mssql@d0d3fc88f84b:/$ printenv PATH
/opt/mssql-tools/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

mssql@d0d3fc88f84b:/$ which healthcheck.sh
/usr/local/bin/healthcheck.sh

mssql@d0d3fc88f84b:/$ healthcheck.sh
mssql@d0d3fc88f84b:/$ echo "Exit code: $?"
Exit code: 0
```

**Comment by thirdgen88** [comment]
Are you sure you're using the new images?  I [moved the HEALTHCHECK](https://github.com/thirdgen88/mssql-docker/blob/3468d5ca09d6477904abc26a8c248ad36e07e0b9/Dockerfile#L25) to be under the path at `/usr/local/bin/healthcheck.sh` instead of `/healthcheck.sh`.

That said, there was still an issue, fixed with #10.

**Comment by alexrudd2** [comment]
> Are you sure you're using the new images? I [moved the HEALTHCHECK](https://github.com/thirdgen88/mssql-docker/blob/3468d5ca09d6477904abc26a8c248ad36e07e0b9/Dockerfile#L25) to be under the path at `/usr/local/bin/healthcheck.sh` instead of `/healthcheck.sh`.

D'oh.  Yes, I was using your new images but your comment made me realize my Docker compose file was the problem.  I still had overridden the healthcheck from when I was layering on top of your image.  (Note the leading `/`)
```yml
healthcheck:
    test: ["CMD", "bash", "/healthcheck.sh"]
```

Works now; thanks.
